prepare-image.sh

```diff
@@ -5,10 +5,10 @@ set -euxo pipefail
 dnf install -y python3 python3-requests
 curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python3 - -b master current-tripleo
 dnf upgrade -y
-dnf --setopt=install_weak_deps=False install -y $(cat /tmp/${PKGS_LIST})
+xargs -rd'\n' dnf --setopt=install_weak_deps=False install -y < /tmp/${PKGS_LIST}
```

Comment:
Can we add a "-t" to the xargs command so that the dnf command that ran is logged (and below)

Reply:
yes, we can! good idea :)